iron-list.html

```diff
@@ -777,6 +831,25 @@
     },
 
     /**
+     * This function will return the amount the physical
+     * top should be incremented based on it's current iterative
+     * state in a given row. This function will only return a 
+     * non-zero value when the index modulous `_physicalItemsPerRow` 
+     * is one less than the possible `_physicalItemsPerRow`
+     */
+    _getPhysicalTopIncrement: function(kth) {
+      if (!Array.isArray(this.items)) return 0;
+      if (kth % this._physicalItemsPerRow !== (this._physicalItemsPerRow - 1)) return 0;
+      var rowSize = 0;
+      for (length = kth + this._physicalItemsPerRow,
+           maxLength = this.items.length;
+           kth < maxLength && kth < length; kth++) {

```

Comment:
Ahh good catch. I need to loop through the items in the current row, not the next.

That should be updated to:

```javascript
var currentRowStart = kth - (kth % this._physicalItemsPerRow);
for (var length = currentRowStart + this._physicalItemsPerRow,
     maxLength = this.items.length;
     kth < maxLength && kth < length; kth++) {
  rowSize = this._physicalSizes[kth] > rowSize ? this._physicalSizes[kth] : rowSize;
}
```